--- a/src/ccip/CCIPLocalSimulatorFork.sol
+++ b/src/ccip/CCIPLocalSimulatorFork.sol
@@ -415,7 +415,7 @@
         }
         Internal.Any2EVMRampMessage memory any2EVMRampMessage = Internal.Any2EVMRampMessage({
             header: message.header,
-            sender: abi.encodePacked(message.sender),
+            sender: abi.encode(message.sender),
             data: message.data,
             receiver: _decodeEVMAddress(message.receiver),
             gasLimit: gasLimit,
